src/libsync/theme.cpp

```diff
@@ -691,6 +691,11 @@ bool Theme::wizardEnableWebfinger() const
     return false;
 }
 
+QVector<QPair<QString, QUrl>> Theme::urlButtons() const
```

Comment:
We should document that a `QVector<QPair...>>` construct is used mainly to preserve the order. Functionally, a map might be superior if the order didn't matter.